extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -59,12 +62,29 @@ export class AmpAdUIHandler {
 
     /** {number} */
     this.state = AdDisplayState.NOT_LAID_OUT;;
+
+    /** {?Element} */
+    this.fallback_ = baseInstance.getFallback();
   }
 
   /**
    * TODO(@zhouyx): Add ad tag to the ad.
    */
   init() {
+    if (!isExperimentOn(this.baseInstance_.win, UX_EXPERIMENT)) {
+      return;
+    }
+
+    if (this.fallback_) {
+      return;
+    }
+    //Apply default fallback div when there's no default one
+    const holder = createElementWithAttributes(document, 'div', {
+      'fallback': '',
+      'layout': 'fill',
```

Comment:
I checked, `layout = fill` won't work here. https://github.com/ampproject/amphtml/blob/master/css/amp.css#L246 already make it layout fill for fallback element. I will remove this.